src/inspector_profiler.cc

```diff
@@ -38,11 +39,11 @@ V8ProfilerConnection::V8ProfilerConnection(Environment* env)
           false)),
       env_(env) {}
 
-uint32_t V8ProfilerConnection::DispatchMessage(const char* method,
+uint64_t V8ProfilerConnection::DispatchMessage(const char* method,
                                                const char* params,
                                                bool is_profile_request) {
   std::stringstream ss;
-  uint32_t id = next_id();
```

Comment:
Can V8 inspector handle uint64 number in JSON?

Reply:
In practice it's very unlikely for the id to go above the maximum of uint32 since it's incremental by the number of requests send. We also don't pass this number to inspector directly but via serialization to JSON. We could add an assertion before we serialize the ID though.

Reply:
Added an assertion.